package.json

```diff
@@ -87,20 +87,20 @@
     "@storybook/jest": "0.2.3",
     "@storybook/test-runner": "0.13.0",
     "@storybook/testing-library": "0.2.2",
-    "@swc-node/register": "~1.6.7",
+    "@swc-node/register": "1.8.0",
     "@swc/core": "~1.3.85",
     "@testing-library/cypress": "9.0.0",
     "@types/decompress": "4.2.7",
     "@types/eslint": "^8.44.2",
     "@types/estree-jsx": "^1.0.3",
     "@types/jest": "^29.4.0",
-    "@types/node": "^20.5.7",
+    "@types/node": "18.19.33",
```

Comment:
Why are we moving down to node v18.19 types?

Reply:
ups forgot to bring it back to where it was... nx migrate did it 🤷‍♂️

Reply:
Brought it to ^20.12.12